desktop/core/src/desktop/settings.py

```diff
@@ -383,6 +383,11 @@
     'default': {
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
         'LOCATION': 'unique-hue'
+    },
+    'hive_discovery': {
```

Comment:
Any chance to define a variable

CACHES_HIVE_DISCOVERY_KEY = 'hive_discovery' to use here and in apps/beeswax/src/beeswax/conf.py 

(to make it a constant)